ee/api/feature_flag_role_access.py

```diff
@@ -16,6 +17,11 @@ class FeatureFlagRoleAccessPermissions(BasePermission):
     def has_permission(self, request, view):
         if request.method in SAFE_METHODS:
             return True
+        if (
```

Comment:
test please 😅 .

These edge cases are so sensitive that when we refactor this in the future (which we for sure will when extending permissions / etc. etc.), it's very easy to forget.

Having a test case preserves knowledge.

Reply:
👍 ✅